packages/bot-strategy-runner/src/index.ts

```diff
@@ -0,0 +1,213 @@
+import program from "commander";
+import nodeFetch from "node-fetch";
+import assert from "assert";
+import * as fs from "fs";
+import * as cliProgress from "cli-progress";
+import * as bluebird from "bluebird";
+
+import { Logger, delay } from "@uma/financial-templates-lib";
+
+import { strategyRunnerConfig, buildBotConfigs, buildGlobalWhitelist, mergeConfig } from "./ConfigBuilder";
+import { runBot } from "./BotEntryWrapper";
+
+// Defaults for bot execution. If the user does not define these then these settings will be used.
+const defaultPollingDelay = 120;
+const defaultStrategyTimeout = 60;
```

Comment:
Should this timeout be 2 mins? I feel like 60 mins is too short, while anything over 2 mins is definitely an error. Empirically speaking.

Reply:
ye, this was mainly to match to the bots implementation. I'm hoary to make this 2 miss and then set the polling delay in to 5 mins. will be easier to reason about I think

Reply:
I've updated to use 5 mins for the polling delay and 2 mins for timeout to be more conservative and consistant with the hub.